tests/unit/MyParks.spec.js

```diff
@@ -0,0 +1,85 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import MyParks from '@/components/MyParks.vue';
+import Vuex from 'vuex';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
```

Comment:
I'm wondering if `localVue` for testing is the equivalent of `Vue.use( )` for implementation.